tools/docker/builder/openwrt/build.sh

```diff
@@ -64,6 +64,12 @@ main() {
                     turris-omnia)
                         TARGET=mvebu
                         ;;
+                    netgear-rax40)
+                        TARGET=intel_mips
+                        PRPLMESH_VARIANT="-dwpal"
+                        PRPL_FEED="https://git.prpl.dev/prplmesh/iwlwav.git^e9ee6b109764dbb91b6cb399ccc0f78e5b04e260"
```

Comment:
Is this really still needed? Can't we still use this feed by specifying the variant?

I'm running a test build, if successful I'll push that with fixup commits.

Reply:
When using the latest feed for the nl80211 targets, dwpal gets compiled although it should not (the intel feed is missing so it fails with safeclibs missing).
I suggest we first merge this PR using the 2 separate feed versions and open an issue about it.
That way we can finally build the dwpal variant in CI and at least spot potential compilation issues introduced by other changes early.